packages/webpack-configs/README.md

```diff
@@ -1,7 +1,22 @@
 # webpack-configs
 
-## Gotchas to remember
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](../../LICENSE)
+[![npm version](https://img.shields.io/npm/v/@workleap/webpack-configs)](https://www.npmjs.com/package/@workleap/webpack-configs)
 
-### Add the @swc/helpers packages to all monorepo imported projects having JSX code
+## Usage
+
+View the [user's documentation](https://gsoft-inc.github.io/wl-web-configs/).
+
+## 🤝 Contributing
+
+View the [contributor's documentation](../../CONTRIBUTING.md).
+
+### Notes
+
+#### Add the @swc/helpers packages to all monorepo imported projects having JSX code
 
 To work properly, all projects having React/JSX code that are referenced by the host project must also install `@swc/helpers` as a `devDependency`.
+
+## License
+
+Copyright © 2023, GSoft inc. This code is licensed under the Apache License, Version 2.0. You may obtain a copy of this license at https://github.com/gsoft-inc/gsoft-license/blob/master/LICENSE.
```

Comment:
```suggestion
Copyright © 2023, Workleap inc. This code is licensed under the Apache License, Version 2.0. You may obtain a copy of this license at https://github.com/gsoft-inc/gsoft-license/blob/master/LICENSE.
```

No?

Reply:
Yes, will reject this one as I did it while fixing the previous comment.